subsys/net/l2/openthread/Kconfig

```diff
@@ -24,6 +32,11 @@ menuconfig NET_L2_OPENTHREAD
 	select MBEDTLS_CIPHER
 	select MBEDTLS_MD
 
+	imply NET_UDP
+	imply NET_SOCKETS
```

Comment:
Shall we imply sockets usage? Simple applications like NCP or CLI does not need this.